### DISCUSSION TITLE
I want to move all emails and their data to Modoba mail server

### DISCUSSION THREAD

**Discussion by 7838099724** [modoboa/modoboa#2283] [General]
I want to move all emails and their data to Modoba mail server how can I do it

# Impacted versions

* OS Type: Debian/Ubuntu,
* OS Version: 18.04
* Database Type:  MySQL
* Database version: X.y
* Modoboa: X.X.x
* installer used: Yes
* Webserver: Nginx
# Steps to reproduce

# Current behavior

# Expected behavior

# Video/Screenshot link (optional)

**Comment by H-A-L-I-S** [comment]
Is that a question or an issue? For me it looks more like something, that should be posted in the `Discussions` section

**Reply by H-A-L-I-S** [reply]
Since it is moved to discussions section here an possible answer: https://github.com/modoboa/modoboa/discussions/2282
